### PR TITLE
Small readability improvements on issue templates

### PR DESCRIPTION
**Pull request by therealmrmuffin**
I have update and adjusted the issue templates to be more informational correct and the template layouts have been tidied up.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new_entry.md [hunk 1]
```diff
@@ -1,32 +1,23 @@
...
-    - `hard`: Sites that require you to contact customer services or those that don't allow automatic or easy account deletion
-    - `impossible`: For sites where it's basically impossible to totally delete your account, even if you contact them
--->
-- `notes`: <!-- *(optional)* Notes will be shown when someone hovers on that service, they include additional information you might need to delete your account -->
-- `email`: <!-- *(optional)* If you have to send an email to a company to cancel your account, add the email address here -->
-- `email_subject`: <!-- *(optional)* Set the subject for the email link. If unset, the default text is "Account Deletion Request" -->
-- `email_body`: <!-- *(optional)* Set the body for the email link. If unset, the default text is "Please delete my account, my username is XXXXXX" -->
-- `domains`: <!-- This is used by the [Chrome extension](https://github.com/jdm-contrib/justdelete.me-chrome-extension) -->
+- `url`: <!-- The url of the account-deletion page. If non exists, the url should be a contact or help page explaining the deletion process. -->
```
Comment (tupaschoal) [inline .github/ISSUE_TEMPLATE/new_entry.md:16]:
```suggestion
- `url`: <!-- The url of the account-deletion page. If none exists, the url should be a contact or help page explaining the deletion process. -->
```

File: .github/ISSUE_TEMPLATE/new_entry.md [hunk 2]
```diff
@@ -1,32 +1,23 @@
...
+- `url_CODE`: <!-- *(optional)* Use the language `CODE` as suffix of the url field to use language-specific deletion codes, shown on the respective language page -->
+- `difficulty`: <!-- Use one of the following difficulties easy/medium/hard/impossible, to learn more about all of the different difficulties [press here](https://github.com/jdm-contrib/justdelete.me-chrome-extension](https://github.com/jdm-contrib/jdm/blob/master/CONTRIBUTING.md). -->
```
Comment (tupaschoal) [inline .github/ISSUE_TEMPLATE/new_entry.md:18]: I don't think this hyperlink works in a markdown comment, did you test it? (even though we had one for the extension already)

File: .github/ISSUE_TEMPLATE/new_entry.md [hunk 3]
```diff
@@ -1,32 +1,23 @@
...
+- `notes`: <!-- Notes will be shown when someone presses the "more info" button on that service's deletion listing. Notes include additional information people might need to delete their accounts. -->
```
Comment (tupaschoal) [inline .github/ISSUE_TEMPLATE/new_entry.md:19]: Why did you remove the optional from notes? We have default phrases for some difficulties, like easy, so it really is optional.

File: .github/ISSUE_TEMPLATE/update_entry.md [hunk 4]
```diff
@@ -1,10 +1,9 @@
 ---
-name: Update Entry
-about: Tell us if you found out that the deletion process changed on a given website
-title: ''
+name: Update Entry.
+about: Tell us if a service you know about has change it deletion process.
```
Comment (tupaschoal) [inline .github/ISSUE_TEMPLATE/update_entry.md:3]:
```suggestion
about: Tell us if a service you know about has changed its deletion process.
```

File: .github/ISSUE_TEMPLATE/update_entry.md [hunk 5]
```diff
@@ -1,10 +1,9 @@
 ---
-name: Update Entry
-about: Tell us if you found out that the deletion process changed on a given website
-title: ''
+name: Update Entry.
```
Comment (tupaschoal) [inline .github/ISSUE_TEMPLATE/update_entry.md:2]:
```suggestion
name: Update Entry
```
No need for that in a title

File: .github/ISSUE_TEMPLATE/update_entry.md [hunk 6]
```diff
@@ -13,11 +12,11 @@ assignees: ''
...
-- [ ] `email`: <!-- New e-mail address -->
-- [ ] `domains`: <!--New e-mail subject -->
+- `name`: <!-- New Name -->
+- `url`: <!-- New Url -->
+- `difficulty`: <!-- New Difficulty: easy/medium/hard/impossible -->
+- `notes`: <!-- New Notes -->
+- `email`: <!-- New Email Address -->
+- `email_subject`: <!-- New Email Subject -->
+- `email_body`: <!-- New Email Body -->
+- `domains`: <!-- New Domains for out [Browser Extensions](https://github.com/jdm-contrib). -->
```
Comment (tupaschoal) [inline .github/ISSUE_TEMPLATE/update_entry.md:22]: Why removing the checkboxes?

File: .github/ISSUE_TEMPLATE/new_entry.md (hunk 1 above)
Comment (therealmrmuffin) [reply .github/ISSUE_TEMPLATE/new_entry.md:16]: Thanks for fixing that.

File: .github/ISSUE_TEMPLATE/new_entry.md (hunk 2 above)
Comment (therealmrmuffin) [reply .github/ISSUE_TEMPLATE/new_entry.md:18]: It works in markdown if i recall correctly.

File: .github/ISSUE_TEMPLATE/new_entry.md (hunk 3 above)
Comment (therealmrmuffin) [reply .github/ISSUE_TEMPLATE/new_entry.md:19]: Didn't know that, honest mistake.

File: .github/ISSUE_TEMPLATE/update_entry.md (hunk 4 above)
Comment (therealmrmuffin) [reply .github/ISSUE_TEMPLATE/update_entry.md:3]: Thanks for correcting me again.

File: .github/ISSUE_TEMPLATE/update_entry.md (hunk 5 above)
Comment (therealmrmuffin) [reply .github/ISSUE_TEMPLATE/update_entry.md:2]: Beg to differ on that, but i can live with it.

File: .github/ISSUE_TEMPLATE/update_entry.md (hunk 6 above)
Comment (therealmrmuffin) [reply .github/ISSUE_TEMPLATE/update_entry.md:22]: Most people seem to remove them, and besides, what's the use case even for them?

File: .github/ISSUE_TEMPLATE/new_entry.md (hunk 3 above)
Comment (tupaschoal) [reply .github/ISSUE_TEMPLATE/new_entry.md:19]:
```suggestion
- `notes`: <!-- *(optional)* Notes will be shown when someone presses the "more info" button on that service's deletion listing. Notes include additional information people might need to delete their accounts. -->
```

File: .github/ISSUE_TEMPLATE/update_entry.md (hunk 6 above)
Comment (tupaschoal) [reply .github/ISSUE_TEMPLATE/update_entry.md:22]: Sounded easier to just check what changed, but I guess leaving it blank has the same meaning, so let's try that out instead then.

There are more complex issue templates now, but I don't have the time to play with them nor I think it will be a great improvement for us here.

File: .github/ISSUE_TEMPLATE/new_entry.md [hunk 7]
```diff
@@ -1,32 +1,23 @@
 ---
 name: New Entry
-about: Tell about a new app, service or website that you found out about the deletion process
-title: ''
+about: Tell us about a new service you've discovered, and how their deletion process works.
+title: 'Add (the service's Name).'
```
Comment (tupaschoal) [inline .github/ISSUE_TEMPLATE/new_entry.md:4]:
```suggestion
about: Tell us about a new service you've discovered, and how their deletion process works
title: 'Add (the service's Name)'
```

File: .github/ISSUE_TEMPLATE/update_entry.md [hunk 8]
```diff
@@ -1,10 +1,9 @@
 ---
 name: Update Entry
-about: Tell us if you found out that the deletion process changed on a given website
-title: ''
+about: Tell us if a service you know about has changed its deletion process.
+title: 'Update (the service's Name).'
```
Comment (tupaschoal) [inline .github/ISSUE_TEMPLATE/update_entry.md:4]:
```suggestion
about: Tell us if a service you know about has changed its deletion process
title: 'Update (the service's Name)'
```

File: .github/ISSUE_TEMPLATE/update_entry.md [hunk 9]
```diff
@@ -13,11 +12,11 @@ assignees: ''
...
-<!-- Check the fields that changed below and provide the new value -->
-
-- [ ] `name`: <!-- New name -->
-- [ ] `url`: <!-- New url -->
-- [ ] `difficulty`: <!-- New difficulty: easy/medium/hard/impossible -->
-- [ ] `notes`: <!-- Updated notes -->
-- [ ] `email`: <!-- New e-mail address -->
-- [ ] `domains`: <!--New e-mail subject -->
+- `name`: <!-- New Name -->
+- `url`: <!-- New Url -->
```
Comment (tupaschoal) [inline .github/ISSUE_TEMPLATE/update_entry.md:16]:
```suggestion
- `url`: <!-- New URL -->
```

**Review by tupaschoal — COMMENTED** [review]
a few other minor changes before merging

File: .github/ISSUE_TEMPLATE/update_entry.md (hunk 6 above)
Comment (therealmrmuffin) [reply .github/ISSUE_TEMPLATE/update_entry.md]: "Don't fix it if it ant broke"

**Comment by therealmrmuffin** [comment]
> a few other minor changes before merging

All good with me.

**Comment by therealmrmuffin** [comment]
@tupaschoal one question i have for you tho is, is there a valid reason for removing all of the periods?

**Comment by tupaschoal** [comment]
I don't think we need the periods in the titles, on the descriptions/about I think I might have gone overboard :P

**Comment by therealmrmuffin** [comment]
> I don't think we need the periods in the titles, on the descriptions/about I think I might have gone overboard :P

You or i? 😅
